tests/test_standard_tags.py

```diff
@@ -110,11 +111,9 @@ def setup_url_with_sensitive_query_string(self):
         ]
 
     @missing_feature(
-        context.library in ["golang", "nodejs", "php", "ruby"],
+        context.library in ["golang", "nodejs", "php", "ruby", "python", "java"],
```

Comment:
@christophe-papazian @ValentinZakharov

Reply:
the new query string is updated in java and released recently